tests/cross/src/test/scala/tests/pc/CompletionBacktickSuite.scala

```diff
@@ -29,7 +29,7 @@ class CompletionBacktickSuite extends BaseCompletionSuite {
   )
 
   check(
-    "comment",
+    "comment".only,
```

Comment:
remove

Reply:
I was pretty sure I also once forgotten to remove `.only` and CI fails if it finds this, super cool feature :)
(but this time I can't find it in logs, weird)

Reply:
Fixed!